db/migrate/20220304145225_create_places.rb

```diff
@@ -0,0 +1,33 @@
+class CreatePlaces < ActiveRecord::Migration[6.1]
+  def change
+    create_table :places do |t|
+      t.string :name, limit: 100, null: false
+      t.integer :accomodation_type_id, index: true
+      t.string :city
+      t.string :region
+      t.integer :rooms_available
+      t.integer :beds
+      t.integer :kids_beds
+      t.boolean :is_pets_allowed
+      t.integer :capacity, default: 2, null: false
+      t.integer :additional_child_place, default: 0
+      t.integer :coordinator_id
+      t.integer :status
```

Comment:
I believe we have PlaceStatus model, probably needs to be a foreign key
@vova is this correct?

Reply:
yes, r u right!